src/main/java/propertyFiles/Bundle.properties

```diff
@@ -11,6 +11,7 @@ file=File
 public=Public
 restricted=Restricted
 restrictedaccess=Restricted Access
+restrictedWithAccessGranted=Restricted with Access Granted
```

Comment:
I checked is only uses as a title on an image lock of a file search card; so let's have consistency and just have the one (restrictedaccess=Restricted with Access Granted)

Reply:
Done in 47ef8f670. Now the search card is consistent with other places:

<img width="417" alt="Screen Shot 2021-04-16 at 11 17 07 AM" src="https://user-images.githubusercontent.com/21006/115047013-1b9d8580-9ea6-11eb-8c57-768ed96ce77a.png">